### PR TITLE
Add node to the PATH for documentation build

### PR DESCRIPTION
**Pull request by arv**

### CODE REVIEW

File: tools/build-js-documentation.py [hunk 1]
```diff
@@ -37,7 +53,9 @@ def main():
         '--output', os.path.join(workspace, 'build', version),
         '--project-version', version
     ]
-    subprocess.check_call(cmd)
+    proc = subprocess.Popen(cmd, env=env)
```
Comment (kalman) [inline tools/build-js-documentation.py]: you could factor this out into a method like `call_with_env(cmd)`

File: tools/build-js-documentation.py [hunk 2]
```diff
@@ -9,22 +9,38 @@
...
+import os
+import subprocess
 
 def main():
     # Workspace is the root of the builder, e.g. "/var/lib/jenkins/workspace/builder".
     workspace = os.getenv('WORKSPACE')
     assert workspace
 
+    # Directory where node/npm binaries have been installed.
+    node_bin = '/var/lib/jenkins/node-v5.11.1-linux-x64/bin'
```
Comment (kalman) [inline tools/build-js-documentation.py]: Every time I see this hardcoded I wonder if we're just creating work for ourselves when we update to node v6. Which, we should do :-\

**Comment by kalman** [comment]
lgtm

File: tools/build-js-documentation.py (hunk 2 above)
Comment (arv) [inline tools/build-js-documentation.py]: I don't understand why we are not adding a symlink in `/usr/local/node/bin` like we have done for go?

Comment (kalman) [inline tools/build-js-documentation.py]: That sounds like a good idea to me...

**Pull request merged**